### DISCUSSION TITLE
Relation presets for interfaces

### DISCUSSION THREAD

**Discussion by SeanDylanGoff** [directus/directus#10997] [Feature Requests]
It seems that while some internal interfaces do this (file, files, translations) there is no obvious way for an interface to "suggest" a relation setup (in my case M2M: target collection name, junction collection sort field, relational trigger setup).
It would be nice if that was possible at some point to save time/reduce mistakes when setting up new fields.

My application a custom image interface that doesn't reference directus_files...

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
